backend/zeno_backend/database/select.py

```diff
@@ -334,6 +327,55 @@ def public_reports() -> list[Report]:
         return reports
 
 
+def project_count(user: User | None = None) -> int:
+    """Get the number of projects.
+
+    Args:
+        user (User | None): the user making the request.
+
+    Returns:
+        int: the number of projects.
+    """
+    db = Database()
+    if user is None:
+        res = db.connect_execute_return(
+            "SELECT COUNT(*) FROM projects WHERE public = TRUE;"
+        )
+    else:
+        res = db.connect_execute_return(
+            sql.SQL("SELECT COUNT(*) FROM") + PROJECTS_BASE_QUERY + sql.SQL(";"),
```

Comment:
This might speak against including likes. Don't know.

Reply:
Yup!